docs/getting-started-guides/mesos.md

```diff
@@ -0,0 +1,303 @@
+
+### Setup Access to Mesos Admin GUI
+This doc will refer to the running Mesos cluster repeatedly.  It's a good idea to make the Mesos admin GUI accessible via the browser.  First, take note of the network name attached to your Mesos cluster (either via the Google Cloud GUI) or with `gcloud` tool as follows:
+
+```bash
+gcloud compute firewall-rules list
+```
+Next, add a firewall-rule to open traffic on port 5050.  This will allow you to access Mesos master GUI via the browser.
+
+```bash
+gcloud compute firewall-rules create mesos-master-gui --network ${network_name}  --source-ranges 0.0.0.0/0 --allow tcp:5050
+```
+Once the rule is applied, point your browser to your Mesos master's IP address (see pre-requisites).  You should see the Mesos GUI displayed.
+
+### Deploy Kubernetes-Mesos
+
+Log into the master node over SSH, replacing the placeholder below with the correct IP address.
+
+```bash
+ssh jclouds@${ip_address_of_master_node}
+```
+
+Build Kubernetes-Mesos.
+
+```bash
+$ git clone https://github.com/mesosphere/kubernetes-mesos k8sm
+$ mkdir -p bin && sudo docker run --rm -v $(pwd)/bin:/target \
+  -v $(pwd)/k8sm:/snapshot -e GIT_BRANCH=release-0.4 \
+  mesosphere/kubernetes-mesos:build
+```
+
+Set some environment variables.
+The internal IP address of the master is visible via the cluster details page on the Mesosphere launchpad, or may be obtained via `hostname -i`.
+
+```bash
+$ export servicehost=$(hostname -i)
+$ export mesos_master=${servicehost}:5050
+$ export KUBERNETES_MASTER=http://${servicehost}:8888
+```
+### Deploy etcd
+Start etcd and verify that it is running:
+
+```bash
+$ sudo docker run -d --hostname $(hostname -f) --name etcd -p 4001:4001 -p 7001:7001 coreos/etcd
+```
+
+```bash
+$ sudo docker ps
+CONTAINER ID   IMAGE                COMMAND   CREATED   STATUS   PORTS                NAMES
+fd7bac9e2301   coreos/etcd:latest   "/etcd"   5s ago    Up 3s    2379/tcp, 2380/...   etcd
+```
+It's also a good idea to ensure your etcd instance is reachable by testing it
+```bash
+curl -L http://$servicehost:4001/v2/keys/
+```
+If connectivity is OK, you will see an output of the available keys in etcd (if any).
+
+### Start Kubernetes-Mesos Services
+Start the kubernetes-mesos API server, controller manager, scheduler, and proxy:
+
+```bash
+$ ./bin/km apiserver \
+  --address=${servicehost} \
+  --mesos_master=${mesos_master} \
+  --etcd_servers=http://${servicehost}:4001 \
+  --portal_net=10.10.10.0/24 \
+  --port=8888 \
+  --cloud_provider=mesos \
+  --v=1 >apiserver.log 2>&1 &
+
+$ ./bin/km controller-manager \
+  --master=$servicehost:8888 \
+  --mesos_master=${mesos_master} \
+  --v=1 >controller.log 2>&1 &
+
+$ ./bin/km scheduler \
+  --address=${servicehost} \
+  --mesos_master=${mesos_master} \
+  --etcd_servers=http://${servicehost}:4001 \
+  --mesos_user=root \
+  --api_servers=$servicehost:8888 \
+  --v=2 >scheduler.log 2>&1 &
+
+$ sudo ./bin/km proxy \
```

Comment:
Why are you running the service proxy on the master? Usually that is only run on the kubelet machines (unless I am forgetting something).